src/h5web/visualizations/nexus/hooks.ts

```diff
@@ -22,27 +22,22 @@ export function useNxData(group: Group): NxData {
 
   const signalDataset = findSignalDataset(group);
   const signalValue = values[signalDataset.name];
-  if (!signalValue) {
-    return { signal: { dims: signalDataset.shape.dims } };
-  }
```

Comment:
No need for the early return as mentioned. If we reach this point, the array must be available.